pysat/instruments/cosmic_gps.py

```diff
@@ -55,8 +57,9 @@
 tags = {'ionprf': '',
         'sonprf': '',
         'wetprf': '',
-        'atmprf': ''}
-sat_ids = {'': ['ionprf', 'sonprf', 'wetprf', 'atmprf']}
+        'atmprf': '',
+        'scnlv1': ''}
+sat_ids = {'': ['ionprf', 'sonprf', 'wetprf', 'atmprf', 'scnlv1']}
```

Comment:
a test date for scintillation should be added. While tests aren't run now, they may be in the future.

Reply:
Thanks for pointing this out, can't believe I missed it.